api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -41,6 +43,14 @@ public Claimant createClaim(Claim claim) {
         }
     }
 
+    private EligibilityStatus determineEligibilityStatusForHousehold(EligibilityResponse eligibilityResponse) {
+        boolean eligibleClaimExistsForHousehold = claimantRepository.eligibleClaimExistsForHousehold(
+                Optional.ofNullable(eligibilityResponse.getDwpHouseholdIdentifier()),
```

Comment:
Rather than wrap this ourselves we could make the identifiers optional in the json object.